modyn/config/schema/pipeline/trigger/drift/detection_window/amount.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+from functools import cached_property
+from typing import Annotated, Literal
+
+from pydantic import Field
+
+from modyn.config.schema.base_model import ModynBaseModel
+from modyn.const.regex import REGEX_TIME_UNIT
+from modyn.utils.utils import SECONDS_PER_UNIT
+
+
+class _BaseWindowingStrategy(ModynBaseModel):
```

Comment:
should we move this to init.py / separate file? seems arbitrary that the base strategy lives in the amount window and the time window has to import it

Reply:
actually, we also have window.py. so this is duplicated